changes.go

```diff
@@ -212,7 +212,7 @@ func (ch *AddServiceChange) GUIArgs() []interface{} {
 	if options == nil {
 		options = make(map[string]interface{}, 0)
 	}
-	return []interface{}{ch.Params.Charm, ch.Params.Service, options}
+	return []interface{}{ch.Params.Charm, ch.Params.Service, options, ch.Params.Constraints}
```

Comment:
In AddUnitChanges.GUIArgs we produce null if To is empty. I guess we could consider that too.